### PR TITLE
Fetch chat gpt 3.5

### PR DESCRIPTION
**Pull request by rccj**

### CODE REVIEW

File: src/pages/type-writer.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from 'react'
```
Comment (rayatn1011) [inline src/pages/type-writer.tsx]: /src/features/type-writer/index.ts
/src/features/type-writer/text-writer.tsx
/src/features/type-writer/markdown-writer.tsx

or one component support both

File: src/pages/type-writer.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+function Typewriter({ content }: { content: string }) {
+  const [text, setText] = useState('')
+
+  useEffect(() => {
+    let i = 0
```
Comment (rayatn1011) [inline src/pages/type-writer.tsx]:
```suggestion
    let textLength = 0
```

File: src/pages/home-page.tsx [hunk 3]
```diff
@@ -12,9 +13,14 @@ function HomePage() {
...
     debounceRef.current = setTimeout(() => {
       const messages = [
-        { role: 'user', content: `請幫我潤飾以下文字 ${inputMessage}` },
+        {
+          role: 'user',
+          content: `請幫我潤飾以下文字 ${inputMessage}`,
+        },
       ]
-      fetchChat(messages)
+      if (inputMessage.trim()) {
```
Comment (rayatn1011) [inline src/pages/home-page.tsx]:
```suggestion
      if (inputMessage.trim().length > 0) {
```

Comment (rccj) [reply src/pages/home-page.tsx]: 5935976cbd4b6ea2010f0eea95e4be82351a53b9

File: src/pages/type-writer.tsx (hunk 2 above)
Comment (rccj) [reply src/pages/type-writer.tsx]: plz check  b64c39069ed0d7153fd557e5152b977ba90a2a0e

File: src/pages/type-writer.tsx (hunk 1 above)
Comment (rccj) [reply src/pages/type-writer.tsx]: respect for god fox !!! 537796e488391e7702889cd1fae99941eea08816

**Review by rayatn1011 — APPROVED** [review]

**Pull request merged**